Fixes error compiling with gcc-3.4.0:

os/os_oflags.c: In function `__db_oflags':
os/os_oflags.c:48: error: label at end of compound statement
make[2]: *** [/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.1.3/build-glibc/db2/os_oflags.os] Error 1

--- glibc-2.1.3/db2/os/os_oflags.c.old	Sun Apr 11 19:56:44 2004
+++ glibc-2.1.3/db2/os/os_oflags.c	Sun Apr 11 19:57:06 2004
@@ -44,7 +44,7 @@
 	case O_RDWR:
 		break;
 	default:		/* Bogus flags value from user.  */
-	  /* XXX no way to return error from here */
+		; /* XXX no way to return error from here */
 	}
 	if (oflags & O_CREAT)
 		dbflags |= DB_CREATE;
